src/base/bittorrent/torrenthandleimpl.cpp

```diff
@@ -1510,8 +1491,6 @@ void TorrentHandleImpl::handleSaveResumeDataAlert(const lt::save_resume_data_ale
 
     if (useDummyResumeData) {
         resumeData["qBt-magnetUri"] = createMagnetURI().toStdString();
-        resumeData["paused"] = isPaused();
-        resumeData["auto_managed"] = isAutoManaged();
```

Comment:
This caused the problem described in https://github.com/qbittorrent/qBittorrent/pull/13123#issuecomment-655716754

Reply:
Thanks! This is it.
Unfortunately there was another error in my code that affected it, so it confused me.